lib/msf/core/exploit/remote/ldap.rb

```diff
@@ -253,42 +253,42 @@ def validate_query_result!(query_result, filter)
       when 1
         # This is unknown as whilst we could fail on lack of authorization, this is not guaranteed with this error code.
         # The user will need to inspect the error message to determine the root cause of the issue.
-        fail_with(Failure::Unknown, "An LDAP operational error occurred on #{filter}. It is likely the client requires authorization! The error was: #{query_result['errorMessage'].strip}")
+        fail_with(Msf::Exploit::Remote::Failure::Unknown, "An LDAP operational error occurred on #{filter}. It is likely the client requires authorization! The error was: #{query_result['errorMessage'].strip}")
```

Comment:
Is there any preference in using `Msf::Exploit::Remote::Failure::Unknown` versus `Msf::Module::Failure::Unknown` like in https://github.com/rapid7/metasploit-framework/pull/17443/files ? 👀

Reply:
Fixed with latest update.